dist/tools/travis-scripts/get-pkg-list.py

```diff
@@ -29,7 +29,7 @@
 known_mcu_groups = arm_mcu_groups + msp_mcu_groups + x86_mcu_groups + \
     avr8_mcu_groups + static_tests_groups
 
-common_pkgs = ["pcregrep", "libpcre3", "python3"]
+common_pkgs = ["pcregrep", "libpcre3", "python3", "python3-pexpect"]
```

Comment:
Well, both the python3 in our Dockerfile and the one that travis is using work fine with this PR.